packages/git/src/browser/git-frontend-module.ts

```diff
@@ -53,4 +56,6 @@ export default new ContainerModule(bind => {
     bind(GitQuickOpenService).toSelf().inSingletonScope();
 
     bind(LabelProviderContribution).to(GitUriLabelProviderContribution).inSingletonScope();
+    bind(GitDecorator).toSelf().inSingletonScope();
```

Comment:
it is redundant

Reply:
I thought inversify (unlike e.g. Guice) does not support implicit bindings. So everything I would like to inject has to be bound. It is clear, that I am not explicitly injecting it, but what if someone would like to do so. Is it still redundant? I really would like this understand. Thanks!

Reply:
> It is clear, that I am not explicitly injecting it, but what if someone would like to do so. Is it still redundant?

then on next line you should use `toService`, otherwise injecting `GitDecorator` and `NavigatorTreeDecorator` will inject 2 different instances